easybuild/easyconfigs/p/PMIx/PMIx-4.1.0-GCCcore-11.2.0.eb

```
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'PMIx'
+version = '4.1.0'
+
+homepage = 'https://pmix.org/'
+description = """Process Management for Exascale Environments
+PMI Exascale (PMIx) represents an attempt to
+provide an extended version of the PMI standard specifically designed
+to support clusters up to and including exascale sizes. The overall
+objective of the project is not to branch the existing pseudo-standard
+definitions - in fact, PMIx fully supports both of the existing PMI-1
+and PMI-2 APIs - but rather to (a) augment and extend those APIs to
+eliminate some current restrictions that impact scalability, and (b)
+provide a reference implementation of the PMI-server that demonstrates
+the desired level of scalability.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/openpmix/openpmix/releases/download/v%(version)s']
+sources = ['%(namelower)s-%(version)s.tar.bz2']
+checksums = ['145f05a6c621bfb3fc434776b615d7e6d53260cc9ba340a01f55b383e07c842e']
+
+builddependencies = [('binutils', '2.37')]
+
+dependencies = [
+    ('libevent', '2.1.12'),
+    ('zlib', '1.2.11'),
+    ('hwloc', '2.5.0'),
+]
+
+configopts = ' --with-libevent=$EBROOTLIBEVENT --with-zlib=$EBROOTZLIB'
+configopts += ' --with-hwloc=$EBROOTHWLOC'
+configopts += ' --enable-pmix-binaries'
+
```

```suggestion
configopts += ' --disable-man-pages'

```

Disable the man page generation, as that requires Pandoc:
```
*** Man page setup
checking if want man pages... yes
checking for pandoc... no
configure: WARNING: *** Could not find a suitable pandoc on your system.
configure: WARNING: *** You need pandoc >=1.12 to build OpenPMIx man pages.
configure: WARNING: *** See pandoc.org.
configure: WARNING: *** If no man pages are needed, rerun configure with option '--disable-man-pages'.
configure: WARNING: *** NOTE: If you are building from a tarball downloaded from the OpenPMIx GitHub repository, you do not need Pandoc
configure: error: Cannot continue
```